vimrc

```diff
@@ -1,3 +1,5 @@
+set encoding=utf-8
+
```

Comment:
According to documentation, `scriptencoding utf-8` could be added below that too, to specify encoding used in _this_ file. However, without `encoding` weird characters are being shown instead of the middle dot, so I found that this line is enough.